src/components/molecules/TeamCard/styles.jsx

```diff
@@ -0,0 +1,52 @@
+import { styled } from 'styled-components';
+
+export const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+  justify-content: center;
+`;
+
+export const Image = styled.img`
+  width: 262.742px;
+  height: 260px;
+  padding-bottom: 26px;
+  border-radius: 16px;
+`;
+
+export const MeetTheTeam = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+export const Links = styled.a`
+  color: black;
+`;
+
+export const Socials = styled.div`
+  display: flex;
+  padding-top: 26px;
+  color: black;
+  align-items: flex-start;
+  gap: 48px;
+  top-margin: 24px;
+`;
+
+export const TeamCards = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-end;
+  justify-content: space-around;
+  column-gap: 100px;
+  row-gap: 120px;
+`;
+
+export const Heading = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 136px;
```

Comment:
It should not be this large margin for smaller screens.

Reply:
Done 7%